after/syntax/sml.vim

```diff
@@ -13,60 +13,69 @@
 "
 "       au! FileType sml setlocal conceallevel=2
 "
-
 " Fix wonky highlighting for => operator
 syn match smlOperator  "=>"
 
 " Also highlight 'NOTE' and 'Note' in comments
 syn keyword  smlTodo contained NOTE Note
 
-" Highlight type variables (i.e., tokens that look like 'xyz)
-syn match smlType "\<'\w\+\>"
```

Comment:
Ah, I see—it matches the ones that _aren't_ Greek concealed. (I didn't notice because I never use those! :-) )

It totally makes sense that causes my patch to fail, because now `'a` matches the `smlType` but only `a` matches the conceal rule (because there's a lookbehind), so the patterns overlap. Easy to fix; I'll just `contains`/`contained` it.